src/utilities.ts

```diff
@@ -423,27 +410,8 @@ export async function computeShortAggregatorAddress(
         | ethers.Wallet
         | ethers.Signer = providers.fuji
 ): Promise<string> {
-    // Get local instance of router contract
-    const router = getRouterContract(version, wallet)
-    
-    const shortAggregatorFactoryAddress = await router.getShortAggregatorFactoryAddress()
-   
-    // Build salt for 
-    
-    const salt = ethers.utils.solidityKeccak256(
-        ["address", "string"],
-        [shortAggregatorFactoryAddress, ticker]
-    )
-
-    // Compute option chain proxy address using 
-    
-    const shortAggregatorAddress = ethers.utils.getCreate2Address(
-        shortAggregatorFactoryAddress,
-        salt,
-        bytecodeHashes.ArrowOptionChainProxy[version]
-    )
-
-    return shortAggregatorAddress
+    const registry = await getRegistryContract(version, wallet)
+   return await registry.getShortAggregatorAddress(ticker)
```

Comment:
Oof the spacing here haha. Please untab and retab it back into place!

Reply:
done